W-18195573-CHtoCH2-App-Migration (#787)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

[NOTE]
If you create a VPN to provision your CloudHub VPC, wait until the newly created VPN status shows as available before migrating your VPC to a CloudHub 2.0 private space. 

> As soon as the upgrade process starts, the upgraded VPC gets locked for any further updates. 

Reviewer: ```suggestion
As soon as the upgrade process starts, the upgraded VPC is locked for any further updates. 
```

> You can cancel the upgrade process to revert to your VPC at any time by clicking *Cancel Upgrade*. In that case, the system deletes the newly created private space, and your VPC becomes unlocked.

Reviewer: ```suggestion
You can cancel the upgrade process to revert to your VPC at any time by clicking *Cancel Upgrade*. In that case, the system deletes the newly created private space, and your VPC is unlocked.
```

After you provision the private space, the *Cancel Upgrade* button no longer shows. However, you can still roll back the upgrade by deleting the private space.

== Next Steps After VPC Upgrade

Following the VPC upgrade, the CloudHub 2.0 private space is fully provisioned, and you can deploy applications to it.

You have two options for migrating your applications:

Self-Service:: Create new versions in the CloudHub 2.0 private space. Then, delete the corresponding application in the CloudHub VPC.

> Runtime Manager Tool:: Alternatively, migrate your applications using the application migration tool. See xref:app-migration.adoc[].

Reviewer: ```suggestion
Runtime Manager Tool:: Migrate your applications using the application migration tool. See xref:app-migration.adoc[].
```

After you migrate all the applications in a VPC, delete the VPC and any associated DLBs to release your CloudHub entitlements. This action doesn’t affect your CloudHub 2.0 private space.

For the final steps after VPC upgrade and CloudHub application migration, see xref:ch-ch2-migration-completion.adoc[].

[NOTE]

> When deploying a new CloudHub 2.0 application, avoid using the CloudHub default domain to prevent issues such as traffic outages.

Reviewer: This seems like a "Consideration". Maybe add to https://github.com/mulesoft/docs-runtime-manager/pull/787/files#:~:text=%3D%3D%3D%20Considerations%20and%20Limitations

> During the migration, you can continue to deploy and manage applications in your VPC to maintain business continuity. After the upgrade, your CloudHub dedicated VPC becomes locked for any changes to the infrastructure, including environment or business group associations, network connections, and internal DNS, to prevent configuration drift. Make any required changes in the newly created private space instead. The upgrade process applies any CloudHub 2.0 configuration changes to the CloudHub VPC. 

Reviewer: ```suggestion
During the migration, you can continue to deploy and manage applications in your VPC to maintain business continuity. To prevent configuration drift, after the upgrade, your CloudHub dedicated VPC is locked for any changes to the infrastructure, including environment or business group associations, network connections, and internal DNS. Make any required changes to the newly created private space. The upgrade process applies any CloudHub 2.0 configuration changes to the CloudHub VPC. 
```

If your CloudHub VPC doesn't have any network connections, and you create new ones in your new CloudHub 2.0 private space after the upgrade, these new connections aren’t reflected in your CloudHub VPC.

> The firewall rules configured in your VPC get copied to the private space. After that, you can make additional changes to the firewall rules in the VPC and the private space. To avoid configuration drift, apply any firewall rule changes in both places throughout the migration.

Reviewer: ```suggestion
The firewall rules configured in your VPC are copied to the private space. After that, you can make additional changes to the firewall rules in the VPC and the private space. To avoid configuration drift, apply any firewall rule changes in both places throughout the migration.
```

[WARNING]

> VPC firewall rules that have ports other than `80`, `443`, and `30500` to `32500` drop during the migration. 

Reviewer: Should this go in Considerations?

Author: The Considerations section refers to the app migration. This warning refers to the VPC upgrade.

== See Also

* xref:app-migration.adoc[]
* xref:ch-ch2-migration-completion.adoc[]

> * xref:cloudhub-2::index.adoc[]

Reviewer: Listed twice.
